pkgs/games/hyperspeedcube/default.nix

```
@@ -0,0 +1,127 @@
+{ cmake
+, alsa-lib
+, atk
+, cairo
+, directx-shader-compiler
+, expat
+, fontconfig
+, freetype
+, gdk-pixbuf
+, glib
+, gtk3
+, harfbuzz
+, mold
+, pango
+, shaderc
+, zlib
+, libxkbcommon
+, libGL
+, wayland
+, wrapGAppsHook
+, xorg
+, vulkan-headers
+, vulkan-loader
+, vulkan-tools
+, vulkan-tools-lunarg
+, vulkan-extension-layer
+, vulkan-validation-layers
+, yq
+, stdenv
+, lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, python3
+, makeWrapper
+}:
+
+# N.b. older versions of cuda toolkit (e.g. 10) do not support newer versions
+# of gcc.  If you need to use cuda10, please override stdenv with gcc8Stdenv
+rustPlatform.buildRustPackage rec {
+  pname = "hyperspeedcube";
+  version = "1.0.6";
+
+  src = fetchFromGitHub {
+    owner = "HactarCE";
+    repo = "Hyperspeedcube";
+    rev = "v${version}";
+    hash = "sha256-FcQuXxVxiyI4hOKS70m62BtZMfN5FzGTLagS+2B3WdY=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+     "key-names-1.5.1" = "sha256-ZO3mlCoUHCaMnxD2vQno6GNx0+JmwZ8OiqpyveiVfvc=";
+    };
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    xorg.libxcb.dev
+    python3
+    makeWrapper
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    alsa-lib
+    atk
+    cairo
+    directx-shader-compiler
+    expat
+    fontconfig
+    freetype
+    gdk-pixbuf
+    glib
+    gtk3
+    harfbuzz
+    mold
+    pango
+    shaderc
+    zlib
+
+    # for execution errors (see https://github.com/emilk/egui/discussions/1587)
+    libxkbcommon
+    libGL
+
+    # WINIT_UNIX_BACKEND=wayland
+    wayland
+
+    # WINIT_UNIX_BACKEND=x11
+    xorg.libXcursor
+    xorg.libXrandr
+    xorg.libXi
+    xorg.libX11
+
+    vulkan-headers
+    vulkan-loader
+    vulkan-tools
+    vulkan-tools-lunarg
+    vulkan-extension-layer
+    vulkan-validation-layers
+
+    yq
+  ];
+  
+  postInstall = ''
+    patchelf \
+      --add-needed ${vulkan-loader}/lib/libvulkan.so.1 \
+      --add-needed ${libGL}/lib/libEGL.so.1 \
+      $out/bin/hyperspeedcube
+    wrapProgram $out/bin/hyperspeedcube --set WAYLAND_DISPLAY "" --set XDG_SESSION_TYPE ""
+    touch $out/bin/nonportable
+  '';
+
+  meta = with lib; {
+    description = ''
```

```suggestion
    longDescription = ''
```

Also, write a short, one-line `description`  according to the guidelines.